qa/suites/rbd/thrash/workloads/rbd_fsx_rate_limit.yaml

```diff
@@ -0,0 +1,9 @@
+tasks:
+- rbd_fsx:
+    clients: [client.0]
+    ops: 6000
+overrides:
+  ceph:
+    conf:
+      client:
+        rbd qos iops limit = 50
```

Comment:
Not valid JSON, should be ```rbd qos iops limit: 50``` (http://qa-proxy.ceph.com/teuthology/jdillaman-2017-11-13_12:10:01-rbd-wip-jd-testing2-distro-basic-smithi/1844450/teuthology.log)

Reply:
Oh, sorry, what a stupid mistake.